client/post-editor/media-modal/detail/detail-fields.jsx

```diff
@@ -354,7 +354,7 @@ export default connect( ( state ) => {
 	const siteId = getSelectedSiteId( state );
 	const isWpcom = ! isJetpackSite( state, siteId );
 	const hasVideoPrivacyFeature =
-		isWpcom && hasActiveSiteFeature( state, siteId, 'videopress-privacy-setting' );
+		isWpcom && siteHasFeature( state, siteId, 'videopress-privacy-setting' );
```

Comment:
class-wpcom-features does not have `videopress-privacy-setting`, so I think this will always return false.